### ISSUE TITLE
Keka 1.1.2 fails on Mac OS X 10.9

### ISSUE THREAD

**Issue by aonez** [aonez/Keka#257]
As reported by @briatte on https://github.com/aonez/Keka/issues/27#issuecomment-411586296, Keka 1.1.2 fails to get access to files.

## Issue

On Mac OS X 10.9 the Keka preferences are not correctly read/stored. Therefore Keka can't get access to files.

- Opening Keka and removing the older file access (In Keka - Preferences - File Access) gives Keka access trough the session
- As soon as Keka is closed, the new access is replaced by the OS, rendering it unusable in the next session

This message appears randomly when launching Keka:

```log
xpcd[XXX]: restored permissions (100600 -> 100700) on /Users/aone/Library/Containers/com.aone.keka/Container.plist
```

This message appears after updating the file access and closing Keka:

```log
cfprefsd[XXX]: CFPreferences detected an inconsistency. An attempt by process XXX to access preferences in 

*****************************************************
Domain: com.aone.keka
User: aone
Container: /Users/aone/Library/Containers/com.aone.keka/Data
Path: /Users/aone/Library/Containers/com.aone.keka/Data/Library/Preferences/com.aone.keka.plist
plist data:<CFPDXPCBuffer: 0x7ff19b408bf0>
shmem index:1022
dirty:1
byHost:0
mode:600
generation:14

 actually resolved to 

*****************************************************
Domain: com.aone.keka
User: aone
Container: (null)
Path: /Users/aone/Library/Preferences/com.aone.keka.plist
plist data:(null)
shmem index:1666
dirty:0
byHost:0
mode:600
generation:0

 Typically this indicates that the process's sandbox profile changed in a way that added or removed a shared-preference-* rule or changed its container path. To avoid overwriting data incorrectly, cfprefsd is disconnecting this client from this source, its preferences will not be saved to disk
```

This issue does not affect on Mac OS X 10.10 and newer.

## Fix

Right now there's no definitive fix. But:

- Updating Mac OS X to 10.11 will fix this issue. All Mac machines running 10.9 can run 10.10 and even 10.11 flawlessly, with all the feature and security improvements.

**Comment by aonez** [comment]
So on Mac OS X 10.9 the preferences are not correctly read/stored. Therefore Keka can get access to files.

- Opening Keka and removing the older file access (In Keka - Preferences - File Access) gives Keka access trough the session
- As soon as Keka is closed, the new access is replaced by the OS, rendering it unusable in the next session

This does not affect on Mac OS X 10.11 and newer. Not tested yet on Mac OS X 10.10.

**Comment by aonez** [comment]
Finally it was a sandbox check using some 10.9 incompatible code. This will be fixed in 1.1.3.
